app/grandchallenge/verifications/admin.py

```diff
@@ -23,14 +23,15 @@ def mark_not_verified(modeladmin, request, queryset):
 class VerificationAdmin(admin.ModelAdmin):
     list_display = (
         "user",
+        "user_info",
         "created",
         "signup_email",
-        "signup_email_activated",
         "email",
         "email_is_verified",
         "is_verified",
         "verified_at",
     )
+    list_select_related = ("user",)
```

Comment:
In `user_info` we access the profile a lot, so this should be `user__user_profile`